### PR TITLE
WIP - Add some documentation for the package

### PR DESCRIPTION
**Pull request by pvillega**
Adding documentation on the packages, while I understand the code better

### CODE REVIEW

**Comment by wfaler** [comment]
Is this ready to merge, or still WIP?

File: main.go [hunk 1]
```diff
@@ -101,22 +158,41 @@ func bootstrap() *cobra.Command {
...
 	}
+
+	addFlags(cmd, &configFile, &terraformVersion)
+
+	return cmd
+}
+
+func addFlags(cmd *cobra.Command, file *string, terraformVersion *string) {
+	cmd.Flags().StringVarP(file, "config.file", "f", "./config.yaml", "OpenPaaS configuration file to use")
+	cmd.Flags().StringVarP(terraformVersion, "terraform.version", "t", "1.4.2", "Terraform version to use")
```
Comment (wfaler) [inline main.go]: I think latest is 1.7.1, can we update it? Think some tests also need updating.

File: config.yaml.example [hunk 2]
```diff
@@ -36,18 +36,19 @@ providers:
...
   allowed_ips:
     - 178.194.90.93/32
     - 10.42.0.0/16
     - 10.43.0.0/16
   provider_settings:
     location: fsn1
-    ssh_keys:
-    - wfaler@chaordic.io
+    ssh_keys: # use Hetzner hcloud CLI tool to retrieve fingerprints: "hcloud ssh-key list"
+    - 70:67:b9:e2:d2:e6:6f:51:82:e4:6d:cc:c6:ff:26:8a
```
Comment (wfaler) [inline config.yaml.example]: If we use Github keys, is there a way to automate it with hcloud CLI?
Foreach Github key:
- Get Github Key
- Check if it is in Hetzner
- If not, add it

File: main.go (hunk 1 above)
Comment (pvillega) [reply main.go]: Done, updated to 1.7.2

File: config.yaml.example (hunk 2 above)
Comment (pvillega) [reply config.yaml.example]: working on it :)

**Review by wfaler — APPROVED** [review]

**Pull request merged**